pkg/features/kube_features.go

```diff
@@ -720,6 +720,12 @@ const (
 	//
 	// Enables the usage of different protocols in the same Service with type=LoadBalancer
 	MixedProtocolLBService featuregate.Feature = "MixedProtocolLBService"
+
+	// owner: @jayunit100
+	// alpha: v1.21
+	//
+	// Labels all namespaces with a default label
+	DefaultNamespaceLabelStamping featuregate.Feature = "DefaultNamespaceLabelStamping"
```

Comment:
I like the feature gate name in the KEP better `NamespaceDefaultLabelName`. Alternatively something like `NamespaceMetadataLabel` seems fine too.